codecov.yml

```diff
@@ -5,11 +5,14 @@ codecov:
 coverage:
   precision: 2
   round: down
-  range: "70...100"
+  range: "60...100"
```

Comment:
I think that saying that code coverage isn't important is like saying, you enjoy writing legacy code.

> Legacy code is code without tests
> \- Michael Feathers' Working Effectively with Legacy Code (ISBN 0-13-117705-2)

Theres two sides to this, you aren't writing tests for yourself, you are writing tests for the next developer that comes along and is trying to add a new feature without breaking existing functionality.